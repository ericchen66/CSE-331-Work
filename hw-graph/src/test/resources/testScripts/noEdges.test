# Test for edges of a childless node

# Creates an empty graph
CreateGraph graph1

# Adds a node to the graph
AddNode graph1 node1

# Lists the children of the node
ListChildren graph1 node1