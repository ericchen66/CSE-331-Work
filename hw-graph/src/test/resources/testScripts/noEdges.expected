# Test for edges of a childless node

# Creates an empty graph
created graph graph1

# Adds a node to the graph
added node node1 to graph1

# Lists the children of the node
the children of node1 in graph1 are: